cartographer/mapping_2d/local_trajectory_builder.cc

```diff
@@ -36,7 +36,8 @@ LocalTrajectoryBuilder::LocalTrajectoryBuilder(
 
 LocalTrajectoryBuilder::~LocalTrajectoryBuilder() {}
 
-sensor::RangeData LocalTrajectoryBuilder::TransformAndFilterRangeData(
+sensor::RangeData
+LocalTrajectoryBuilder::TransformToGravityAlignedFrameAndFilter(
     const transform::Rigid3f& gravity_alignment,
```

Comment:
I wonder whether we should not rename this argument still. It is a transform from the tracking frame in which the range data was accumulated and to gravity aligned frame at a different time. Not sure what a good name is though. Ideas?

Reply:
The Rigid3f argument transforms from first_pose_estimate_ to the gravity-aligned frame. Let's call it first_tracking_frame_to_gravity_aligned_frame. What do you think?

Reply:
I would prefer a shorter name especially since your suggested name while very long does not convey the most important aspect: As pointed out in the terminology documentation there are separate gravity-aligned frames for different nodes. The one we use here is related to the tracking frame at a later time than the first tracking frame. Maybe `transform_to_gravity_aligned_frame`?